src/ruby.ts

```diff
@@ -284,7 +295,7 @@ export class Ruby {
 
   private async toolExists(tool: string) {
     try {
-      await asyncExec(`${this.shell} -lic '${tool} --version'`);
+      await asyncExec(`${this.shell} ${this.shell_opt_exist} '${tool} --version'`);
```

Comment:
Here, we actually forgot to remove the `-l` flag. It should also just be `-ic` similar to the Ruby activation command.